VAST_INTEGRATION_GUIDE.md

```
@@ -0,0 +1,443 @@
+
+### Enable Debug Logging
+
+```swift
+VideoAdConfiguration(
+    debugLogging: true
+)
+```
+
+This will output detailed logs:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)